zerver/openapi/zulip.yaml

```diff
@@ -15866,17 +15866,39 @@ paths:
                     (This may be significantly slower in organizations with
                     thousands of users subscribed to many channels.)
 
+                    If a client passes `partial` for this parameter, the server may,
+                    for some channels, return a subset of the channel's subscribers
+                    in the `partial_subscribers` field instead of the `subscribers` field,
+                    which always contains the complete set of subscribers. This behavior
+                    is strongly recommended for clients that need subscriber data,
+                    especially in large organizations with tens of thousands of users.
+
+                    The server guarantees that it will always return a `subscribers`
+                    field for channels with fewer than 250 total subscribers. When
+                    returning a `partial_subscribers` field, the server guarantees
+                    that all bot users and users active within the last 30 days will
```

Comment:
So I checked, and `long_term_idle` uses 28 days as the definition. I made the docs say 14 -- I don't think we need to guarantee more than that.